rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -3653,6 +3653,10 @@ public NewClass withArguments(@Nullable JContainer<Expression> arguments) {
         @With
         Expression expression;
 
+        @With
+        @Nullable
```

Comment:
This should not be nullable. 
With collection types the concept of "missing" is best represented with an empty collection, rather than null.

Now, you might say to me "But Sam, `CompilationUnitTree.getPackageAnnotations()` can return `null`, so making this `@Nullable` is consistent with the javac AST". 
To which I hypothetically reply: The documentation for `CompilationUnitTree.getPackageAnnotations()` specifies that it would only return `null` if there was no package declaration at all. That's a helper method anyway, `PackageTree.getAnnotations()` is _not_ nullable. 
```java
    /**
     * Returns the annotations listed on any package declaration
     * at the head of this compilation unit, or {@code null} if there
     * is no package declaration.
     * @return the package annotations
     */
    List<? extends AnnotationTree> getPackageAnnotations();
```

So I feel pretty confident that making this field not nullable is consistent with best practices, the rest of our code, _and_ javac's representation.

Reply:
Thanks, @sambsnyd. I agree and will make the change